### ISSUE TITLE
Adding Bombe to patch causes crash

### ISSUE THREAD

**Issue by alxdb** [jhoar/AmalgamatedHarmonics#33]
As the title says, adding Bombe to patch causes vcvrack to crash.

VCV Rack version 1.1.4, Amalgamated Harmonics version 1.0.1 on Arch Linux.

here is the stack trace printed to `~/.Rack/log.txt` relating to the crash
```
[3.984 fatal src/main.cpp:38] Fatal signal 6. Stack trace:
25: ./Rack() [0x56b921]
24: /usr/lib/libc.so.6(+0x3a7e0) [0x7fde9d2e77e0]
23: /usr/lib/libc.so.6(gsignal+0x145) [0x7fde9d2e7755]
22: /usr/lib/libc.so.6(abort+0x125) [0x7fde9d2d2851]
21: /usr/lib/libstdc++.so.6(+0x9581f) [0x7fde9d66781f]
20: /usr/lib/libstdc++.so.6(+0xa230a) [0x7fde9d67430a]
19: /usr/lib/libstdc++.so.6(+0xa2367) [0x7fde9d674367]
18: /usr/lib/libstdc++.so.6(+0xa25bd) [0x7fde9d6745bd]
17: /usr/lib/libstdc++.so.6(_ZSt20__throw_length_errorPKc+0x45) [0x7fde9d66a3a3]
16: /usr/lib/libstdc++.so.6(+0x141fde) [0x7fde9d713fde]
15: /usr/lib/libstdc++.so.6(_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE9_M_assignERKS4_+0xb0) [0x7fde9d7143c0]
14: /home/alxdb/.Rack/plugins-v1/AmalgamatedHarmonics/plugin.so(_ZN12BombeDisplay4drawERKN4rack6widget6Widget8DrawArgsE+0x164) [0x7fde9a93da92]
13: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
12: ./Rack(_ZN4rack3app12ModuleWidget4drawERKNS_6widget6Widget8DrawArgsE+0x5b) [0x5a0b3b]
11: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
10: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
9: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
8: ./Rack(_ZN4rack6widget10ZoomWidget4drawERKNS0_6Widget8DrawArgsE+0x9a) [0x5b1c04]
7: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
6: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
5: ./Rack(_ZN4rack2ui12ScrollWidget4drawERKNS_6widget6Widget8DrawArgsE+0x33) [0x57375d]
4: ./Rack(_ZN4rack6widget6Widget4drawERKNS1_8DrawArgsE+0x1ce) [0x5b2bfa]
3: ./Rack(_ZN4rack6Window3runEv+0x6cb) [0x564cad]
2: ./Rack(main+0x45a) [0x4e525a]
1: /usr/lib/libc.so.6(__libc_start_main+0xf3) [0x7fde9d2d3ee3]
0: ./Rack(_start+0x29) [0x4ea189]
```

**Comment by jhoar** [comment]
Hi,

Thanks for the report. I can't immediately reproduce it on my Windows system, so I'm going to try and pick through the stack trace and work out what's going on.

cheers,

john

**Comment by the1andonlydrno** [comment]
Hi John, I also can confirm this crash, do think you can fix the bug soon, because I want to use all your modules in VCP 40. If not, I have to leave Bomb out. Thanks for all your efforts, Norbert

**Comment by jhoar** [comment]
H aall, 

Sorry for being a bit slow on this - I have been deeply submerged in getting Prism to work with less CPU and it's also occupying 100% of my personal CPU. :)

I've tracked it down and it is fixed in 1.0.3, and submitted for release by the Plugin team. Should be out in the next day or so.

**Comment by alxdb** [comment]
cheers @jhoar 👍. Also, thanks for these great modules!

**Comment by jhoar** [comment]
Closed in 1.0.3
